lib/node_modules/@stdlib/blas/ext/base/dsnansum/test/test.dsnansum.js

```diff
@@ -142,18 +141,18 @@ tape( 'the function supports a `stride` parameter', function test( t ) {
 		NaN
 	]);
 
-	N = floor( x.length / 2 );
+	N = 4;
 	v = dsnansum( N, x, 2 );
```

Comment:
```suggestion
	v = dsnansum( 4, x, 2 );
```

Apply this in every `test.*` file